helm/cas-ggircs/values-dev.yaml

```
@@ -32,3 +32,7 @@ persistence:
 
 nginx-sidecar:
   hostName: cas-ggircs-dev.apps.silver.devops.gov.bc.ca
+
+devops:
+  terraform:
+   namespace_apps: '["ggircs-backups", "eccc-swrs-import", "mb-backups"]'
```

`mb-backups` is for metabase, which is its own chart/repository: https://github.com/bcgov/cas-metabase
It also configures a separate instance of `cas-postgres`, with its own backup buckets: https://github.com/bcgov/cas-metabase/blob/develop/helm/cas-metabase/values-dev.yaml

```suggestion
   namespace_apps: '["ggircs-backups", "eccc-swrs-import"]'
```

---

I'll go in and re-import the state so it doesn't include _`mb-backups`_. Then we'll need to do a migration on `cas-metabase` for it to own it's own state.